embedded/servo_driver/servo_test.cc

```diff
@@ -1,15 +1,44 @@
 #include "servo_driver.hh"
```

Comment:
Use the full path `embedded/servo_driver/servo_test.cc` so it can be found by pymake